setup.py

```diff
@@ -50,65 +49,72 @@ def cmake(self):
         cmake_cmd = shutil.which("cmake")
         return cmake_cmd
 
+    @property
+    def make(self):
+        """Find and use installed cmake"""
+        make_cmd = shutil.which("make")
+        return make_cmd
+
     def run(self):
+        # Validate dependencies
+        check_prereq("cmake")
         check_prereq("make")
         check_prereq("gcc")
         check_prereq("g++")
 
+        # Set up parameters
+        source_directory = Path(os.path.abspath(os.path.dirname(__file__))).resolve()
```

Comment:
Super minor, but if you want to make this a hair simpler, this should be the equivalent:
```py
source_directory = Path(__file__).parent.resolve()
```